app/views/declaration/summary/summary_page.scala.html

```diff
@@ -74,14 +80,20 @@
 
     @items_section(suppDecData.items)
 
-    @helper.form(action = SummaryController.submitSupplementaryDeclaration(), 'id -> "summary") {
-        @helper.CSRF.formField
+    @if(mode == Mode.AmendMode) {
```

Comment:
@ed0906 what you think about 
```
mode.onAmend {
  <a class="button mt-3" href="@routes.DispatchLocationController.displayPage()">@messages("site.continue")</a>
}
```

Reply:
Hmmmm, I don't think its what we would intuitively think to use

Reply:
It same concept like foreach on `Option` but for enumeration